flashlight: various tweaks

RobotCaleb opened the pull request:
@bwidawsk I made some changes to get it working. If I did this right I should be requesting to merge engineers_nightmare/flashlight_caleb into your flashlight branch.

5m was not quite far enough. It made the flashlight feel really weird.
I disabled rendering on flashlight entity type. This will prevent a mesh getting drawn and (maybe prevent) inserted into physics sim.
I added a reader to the entity to remove a runtime exception.
I removed the bogus type as we removed that feature.

Feel free to take this, or to use it as inspiration.

Nice job on the flashlight. I like it.

bwidawsk commented:
Thanks! I'd really like to get this merged so I can avoid the various breakages
we saw on my first pull request. Any thoughts on that? Do you want me to keep
working on it out of tree?

On Tue, Oct 13, 2015 at 10:23:29AM -0700, Caleb Anderson wrote:

> @bwidawsk I made some changes to get it working. If I did this right I should be requesting to merge engineers_nightmare/flashlight_caleb into your flashlight branch.
> 
> 5m was not quite far enough. It made the flashlight feel really weird.
> I disabled rendering on flashlight entity type. This will prevent a mesh getting drawn and (maybe prevent) inserted into physics sim.
> I added a reader to the entity to remove a runtime exception.
> I removed the bogus type as we removed that feature.
> 
> Feel free to take this, or to use it as inspiration.
> 
> Nice job on the flashlight. I like it.
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/bwidawsk/engineers-nightmare/pull/1
> 
> -- Commit Summary --
> - flashlight: various tweaks
> 
> -- File Changes --
> 
> ```
> M main.cc (22)
> ```
> 
> -- Patch Links --
> 
> https://github.com/bwidawsk/engineers-nightmare/pull/1.patch
> https://github.com/bwidawsk/engineers-nightmare/pull/1.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/bwidawsk/engineers-nightmare/pull/1

RobotCaleb commented:
You can take this into your branch and get the PR update, or I can submit a
PR for this if you don't think anymore changes need to happen.

On Tue, Oct 13, 2015 at 11:36 AM Ben Widawsky notifications@github.com
wrote:

> Thanks! I'd really like to get this merged so I can avoid the various
> breakages
> we saw on my first pull request. Any thoughts on that? Do you want me to
> keep
> working on it out of tree?
> 
> On Tue, Oct 13, 2015 at 10:23:29AM -0700, Caleb Anderson wrote:
> 
> > @bwidawsk I made some changes to get it working. If I did this right I
> > should be requesting to merge engineers_nightmare/flashlight_caleb into
> > your flashlight branch.
> > 
> > 5m was not quite far enough. It made the flashlight feel really weird.
> > I disabled rendering on flashlight entity type. This will prevent a mesh
> > getting drawn and (maybe prevent) inserted into physics sim.
> > I added a reader to the entity to remove a runtime exception.
> > I removed the bogus type as we removed that feature.
> > 
> > Feel free to take this, or to use it as inspiration.
> > 
> > Nice job on the flashlight. I like it.
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/bwidawsk/engineers-nightmare/pull/1
> > 
> > -- Commit Summary --
> > - flashlight: various tweaks
> > 
> > -- File Changes --
> > 
> > M main.cc (22)
> > 
> > -- Patch Links --
> > 
> > https://github.com/bwidawsk/engineers-nightmare/pull/1.patch
> > https://github.com/bwidawsk/engineers-nightmare/pull/1.diff
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/bwidawsk/engineers-nightmare/pull/1
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bwidawsk/engineers-nightmare/pull/1#issuecomment-147787406
> .

bwidawsk commented:
I'll give it a whirl on my commute home and let you know. I just wanted to find
out the general plan.

Thanks.

On Tue, Oct 13, 2015 at 10:45:10AM -0700, Caleb Anderson wrote:

> You can take this into your branch and get the PR update, or I can submit a
> PR for this if you don't think anymore changes need to happen.
> 
> On Tue, Oct 13, 2015 at 11:36 AM Ben Widawsky notifications@github.com
> wrote:
> 
> > Thanks! I'd really like to get this merged so I can avoid the various
> > breakages
> > we saw on my first pull request. Any thoughts on that? Do you want me to
> > keep
> > working on it out of tree?
> > 
> > On Tue, Oct 13, 2015 at 10:23:29AM -0700, Caleb Anderson wrote:
> > 
> > > @bwidawsk I made some changes to get it working. If I did this right I
> > > should be requesting to merge engineers_nightmare/flashlight_caleb into
> > > your flashlight branch.
> > > 
> > > 5m was not quite far enough. It made the flashlight feel really weird.
> > > I disabled rendering on flashlight entity type. This will prevent a mesh
> > > getting drawn and (maybe prevent) inserted into physics sim.
> > > I added a reader to the entity to remove a runtime exception.
> > > I removed the bogus type as we removed that feature.
> > > 
> > > Feel free to take this, or to use it as inspiration.
> > > 
> > > Nice job on the flashlight. I like it.
> > > You can view, comment on, or merge this pull request online at:
> > > 
> > > https://github.com/bwidawsk/engineers-nightmare/pull/1
> > > 
> > > -- Commit Summary --
> > > - flashlight: various tweaks
> > > 
> > > -- File Changes --
> > > 
> > > M main.cc (22)
> > > 
> > > -- Patch Links --
> > > 
> > > https://github.com/bwidawsk/engineers-nightmare/pull/1.patch
> > > https://github.com/bwidawsk/engineers-nightmare/pull/1.diff
> > > 
> > > ---
> > > 
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/bwidawsk/engineers-nightmare/pull/1
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/bwidawsk/engineers-nightmare/pull/1#issuecomment-147787406
> > .
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/bwidawsk/engineers-nightmare/pull/1#issuecomment-147790873
